app/(candidate)/dashboard/contacts/[[...attr]]/components/ContactSearch.js

```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import { Input } from 'goodparty-styleguide'
+import { useState } from 'react'
+
+export const ContactSearch = () => {
+  const [searchText, setSearchText] = useState('')
+
+  // TODO in follow-up:
+  // - add a search icon
+  // - hook up to API
+  return (
+    <Input
+      className="max-w-md"
+      placeholder="Search contacts by name or phone number"
+      value={searchText}
+      onChange={(e) => {
+        setSearchText(e.target.value)
+      }}
+    />
+  )
+}
```

Comment:
Instead of creating another search input, seems like maybe this should just be leveraged? 🤔 

https://github.com/thegoodparty/gp-webapp/blob/29a71af590c59e0871889c4bf2d67241b82b653d/app/(candidate)/dashboard/issues/components/search/SearchContainer.js#L37-L48

Reply:
@RavenHursT Thanks for pointing to me to that, will do. Okay if I merge this change, and follow-up to use the shared SearchInput + make it a touch more extensible? As-is, I noticed a couple problems with the `SearchInput`:
- hard-coded `placeholder` text (easy fix)
- it doesn't seem to use the brand font -- it's just using the default browser font (slightly bigger fix -- I think we might need an mui provider change or something)